http-clients/build.gradle

```diff
@@ -21,6 +21,7 @@ dependencies {
     compile libs.wsRsApi
 
     testCompile project(':http-servers')
+    testCompile files('lib/VersionTest.jar')
```

Comment:
Not a fan of generating since it makes running tests a multistep process. regexing the user agent generated using something like Feign could work, just seems brittle since we're depending on the format a non-public non-static string